server.py

```diff
@@ -1,61 +1,76 @@
-SOCKET_HOST = os.getenv('SOCKET_HOST', default='0.0.0.0')
-SOCKET_PORT = int(os.getenv('SOCKET_PORT', default=8888))
-LOG_LEVEL = os.getenv('LOG_LEVEL', default='ERROR')
-REFRESH_RATE = int(os.getenv('REFRESH_RATE', default=5))
+RCON_PORT = os.getenv('RCON_PORT', 25575)
+SOCKET_HOST = os.getenv('SOCKET_HOST', '0.0.0.0')
+SOCKET_PORT = int(os.getenv('SOCKET_PORT', 8888))
+LOG_LEVEL = os.getenv('LOG_LEVEL', 'ERROR')
+REFRESH_RATE = int(os.getenv('REFRESH_RATE', 5))
 
 logging.basicConfig(level=LOG_LEVEL)
 
 assert RCON_HOST is not None
 assert RCON_PASSWORD is not None
 
-async def sendLocations(websocket, path):
+
+async def send_locations(websocket, path):
     while True:
-        output = json.dumps(getLocations())
+        output = json.dumps(get_locations())
         try:
             await websocket.send(output)
-        except (websockets.exceptions.ConnectionClosedOK, websockets.exceptions.ConnectionClosedError, websockets.exceptions.ConnectionClosed):
+        except (websockets.exceptions.ConnectionClosedOK, websockets.exceptions.ConnectionClosedError,
+                websockets.exceptions.ConnectionClosed):
             await websocket.wait_closed()
-            logging.info("Client connection closed")
+            logging.info('Client connection closed')
             break
         finally:
             await asyncio.sleep(REFRESH_RATE)
 
-def getLocations():
+
+def get_locations():
     output = []
-    response = mcr.command("/list")
-    match = re.search(r"([0-9]+).+:(.+)", response)
-    numberOfPlayers = int(match.group(1))
+    response = mcr.command('/list')
+    match = re.search(r'([0-9]+).+:(.+)', response)
+    number_of_players = int(match.group(1))
 
-    if numberOfPlayers != 0:
-        playerList = match.group(2).strip().split(", ")
-        for playerName in playerList:
-            response = mcr.command("/data get entity {} Pos".format(playerName))
-            match = re.search(r"(\[.*\])", response)
-            playerPosition = eval(match.group(1).replace("d",""))
+    if number_of_players != 0:
+        player_list = match.group(2).strip().split(", ")
+        for playerName in player_list:
+            response = mcr.command(f"/data get entity {playerName} Pos")
+            match = re.search(r'(\[.*\])', response)
+            player_position = eval(match.group(1).replace('d', ''))
 
-            response = mcr.command("/data get entity {} Dimension".format(playerName))
-            match = re.search(r"data: (.*)", response)
-            playerDimension = match.group(1).strip("\"")
+            response = mcr.command(f'/data get entity {playerName} Dimension')
+            match = re.search(r'data: (.*)', response)
+            player_dimension = match.group(1).strip("\"")
 
-            output.append({"name": playerName, "position": {"x": playerPosition[0], "y": playerPosition[1], "z": playerPosition[2]}, "dimension": playerDimension})
+            output.append({'name': playerName, 'position': {
```

Comment:
That would be an allowed style too. I do think your suggestion would be more readable. 

Another thing done in Python is often to not align with the brace above, but to leave it on the last line.

> The closing brace/bracket/parenthesis on multiline constructs may either line up under the first non-whitespace character of the last line of list, as in:
```
my_list = [
    1, 2, 3,
    4, 5, 6,
    ]
result = some_function_that_takes_arguments(
    'a', 'b', 'c',
    'd', 'e', 'f',
    )
```
> or it may be lined up under the first character of the line that starts the multiline construct, as in:
```
my_list = [
    1, 2, 3,
    4, 5, 6,
]
result = some_function_that_takes_arguments(
    'a', 'b', 'c',
    'd', 'e', 'f',
)
```